vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -12,11 +12,14 @@
 
 
 class BaseOpenVINOBackend(BaseBackend):
-    def __init__(self, model_xml: os.PathLike, weights_bin: os.PathLike,
+    def __init__(self, model: bytes,
```

Comment:
All OpenVINO models seems to come in the format of ".xml" file and ".bin" file. It might be helpful to name these to reflect that. Something like "model_xml_bytes" and "weights_bin_bytes"?

Reply:
Or maybe just more terse: xml_bytes, bin_bytes

Reply:
Sure! How about just the original names `model_xml` and `weights_bin` since the type annotations tell you it's bytes?

Reply:
That sounds good to me! Next time we work on the Tensorflow backends, we might want to change the name from model_bytes/metadata_bytes to model_pb,metadata_json or something.

Reply:
Okay, I've done that thing!